src/stories/components/UserGreeting/UserGreeting.js

```diff
@@ -0,0 +1,54 @@
+import './styles.scss';
+
+import React, { useState, useEffect, useContext } from 'react';
+import UserServices from '../../../firebase/CRUDServices/userServices';
+
+export const UserContext = React.createContext();
+
+/**
+ * Component for user's greeting element.
+ *
+ * @component
+ * @return {object} (
+ *   <UserGreeting/>
+ * )
+ */
+export const UserGreeting = () => {
+  const service = UserServices.getInstance();
+  const user = useContext(UserContext);
+  const [logInState, setLogInState] = useState({ isLoggedIn: false });
+  const [firstName, setFirstName] = useState('');
+  const [lastName, setLastName] = useState('');
+  const [error, setError] = useState('');
+
+  useEffect(async () => {
+    service.getCurrentUser(setLogInState).then((response) => {
+      if (response.errorCode) {
+        // ERROR HANDLING
+        setLogInState({ isLoggedIn: false });
+        setError(response.errorMessage);
+      } else {
+        setError('');
+        setLogInState({ isLoggedIn: true });
+        setFirstName(user.firstName);
+        setLastName(user.lastName);
+      }
+    });
+  }, [logInState.isLoggedIn]);
+
+  return (
+    <div>
+      { logInState.isLoggedIn ?
+        <UserContext.Provider value={logInState.user}>
+          <div>
+            <h5>Hello, {firstName + ' ' + lastName}</h5>
+          </div>
+        </UserContext.Provider> :
+        <div>
+          { error }
+          <p>No user is logged in</p>
```

Comment:
In my opinion, the page of the website would look better if we don't show this when no user is logged in. I would say it would be better to display a link to the Login page instead. Or for now, to keep it simple we could just not display anything when no one is logged in. But that's just my guess, maybe we could check with @alecherryy or @cmutuc25?

Reply:
I think I'll just show the error message if no user is found for now. When it is integrated with the page, I think it will be handled differently, such as not redirected to the page when login is not authenticated.